op-challenger/game/fault/trace/alphabet/provider.go

```diff
@@ -50,12 +50,13 @@ func (ap *AlphabetTraceProvider) GetStepData(ctx context.Context, pos types.Posi
 		return absolutePrestate, []byte{}, preimageData, nil
 	}
 	// We want the pre-state which is the value prior to the one requested
-	prestateTraceIndex := posIndex.Sub(posIndex, big.NewInt(1))
+	prestateTraceIndex := new(big.Int).Sub(posIndex, big.NewInt(1))
```

Comment:
Ah nice catch, I had fixed this up too in https://github.com/ethereum-optimism/optimism/pull/8853 😄

Reply:
I did try just copying the code from #8853 and it still didn't work. I may have just missed something but I think it was actually an off by two error with the need to do the initial `step` to create the real absolute prestate. With the unit test added it should easy to know when we do and don't have the right step data now though.